can/interfaces/vector/canlib.py

```diff
@@ -233,6 +233,8 @@ def __init__(
         )
 
         if permission_mask.value == self.mask:
+            self._init_access = True
```

Comment:
I'm not a huge fan of this flag but I would rather not check for init access everytime.